Tutorials/Inverese_Kinematics_with_IMUs/imuInverseKinematics_Setup.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<OpenSimDocument Version="40000">
+	<IMUInverseKinematicsTool>
+		<!--Name of the directory where results are written. Be default this is the directory in which the setup file is be  executed.-->
+		<results_directory>C:\Users\Ayman-NMBL\Documents\OpenSim\4.2-2020-11-09-621c2af\Models\Rajagopal_OpenSense</results_directory>
```

Comment:
Probably don't want to hard code your directory here :)

Reply:
Yes, removed.